service/src/main/resources/quips/insultibot/insults.json

```diff
@@ -944,5 +955,599 @@
+  },
+  {
+    "quipGroup":"KICKBALL",
+    "text":"I bet you were always picked last for the kickball team.",
+    "ssml":"I bet you were always picked last for the kickball team.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"I bet [TARGET] was always picked last for the kickball team.",
+    "targetableSsml":"I bet [TARGET] was always picked last for the kickball team."
+  },
+  {
+    "quipGroup":"NOT_ATTRACTIVE",
+    "text":"I'm not sure there's enough alcohol in the world to make you look attractive.",
+    "ssml":"I'm not sure there's enough alcohol in the world to make you look attractive.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"I'm not sure there's enough alcohol in the world to make [TARGET] look attractive.",
+    "targetableSsml":"I'm not sure there's enough alcohol in the world to make [TARGET] look attractive."
+  },
+  {
+    "quipGroup":"NOT_ATTRACTIVE",
+    "text":"You couldn't paint on make-up thick enough to make you look attractive.",
+    "ssml":"You couldn't paint on make-up thick enough to make you look attractive.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"You couldn't paint on make-up thick enough to make [TARGET] look attractive.",
+    "targetableSsml":"You couldn't paint on make-up thick enough to make [TARGET] look attractive."
+  },
+  {
+    "quipGroup":"NOT_ATTRACTIVE",
+    "text":"There's no photoshop artist good enough to make you look attractive.",
+    "ssml":"There's no photoshop artist good enough to make you look attractive.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"There's no photoshop artist good enough to make [TARGET] look attractive.",
+    "targetableSsml":"There's no photoshop artist good enough to make [TARGET] look attractive."
+  },
+  {
+    "quipGroup":"DINOSAUR_BRAIN",
+    "text":"Isn't it exciting to know how much you have in common with dinosaurs? Like your walnut sized brain.",
+    "ssml":"Isn't it exciting to know how much you have in common with dinosaurs? Like your walnut sized brain.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"Isn't it exciting to know how much [TARGET] has in common with dinosaurs? Like a walnut sized brain.",
+    "targetableSsml":"Isn't it exciting to know how much [TARGET] has in common with dinosaurs? Like a walnut sized brain."
+  },
+  {
+    "quipGroup":"MONSTER_UNDER_THE_BED",
+    "text":"Monsters like you are supposed to sleep under the bed, not on it.",
+    "ssml":"Monsters like you are supposed to sleep under the bed, not on it.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"Monsters like [TARGET] are supposed to sleep under the bed, not on it.",
+    "targetableSsml":"Monsters like [TARGET] are supposed to sleep under the bed, not on it."
+  },
+  {
+    "quipGroup":"EACH_DAY_BETTER_THAN_NEXT",
+    "text":"Each day with you is better than the next.",
```

Comment:
What's not understandable about it? The time spent with you keeps getting worse.